plugin/xmark.vim

```diff
@@ -103,7 +103,7 @@ function! s:xmark(resize, bang)
     execute 'augroup!' grp
     if has_key(s:tmp, bufnr('%'))
       let tmp = remove(s:tmp, bufnr('%'))
-      silent! call system(s:render('close', { 'app': s:app, 'outurl': s:urlencode(tmp) }))
+      silent! call system(s:render('close', { 'app': s:app, outurl': s:urlencode(tmp) }))
```

Comment:
Missing single-quote here.
Could you squash the commits into one so that it's easier to see/track changes?